thirdparty/lua-ljsqlite3/init.lua

```diff
@@ -342,10 +376,9 @@ end
 
 -- Connection exec, __call, rowexec --------------------------------------------
 function conn_mt:exec(commands, get) T_open(self)
-  local cmd1 = split(commands, ";")
   local res, n
-  for i=1,#cmd1 do
-    local cmd = trim(cmd1[i])
+  for command in commands:gsplit(";", true) do
```

Comment:
No issue with such SQL ?:
`select * from book where title like '%;%'; select * from book where title = ';';`
(that might probably not happen with out code, but well :)

Reply:
I expect that this will implode in fun and interesting ways, yeah, and that stuff like that might have been what the first fork was trying to fix, let me check ;).

Reply:
(Sidebar: xsys was also blowing up on that)

Reply:
```
split_sql
1       SELECT * FROM book WHERE title LIKE '%;%';
2
SELECT * FROM book WHERE title = ';';
3

xsys
1       SELECT * FROM book WHERE title LIKE '%
2       %'
3
SELECT * FROM book WHERE title = '
4       '
5

Cosmin
1       SELECT * FROM book WHERE title LIKE '%
2       %'
3
SELECT * FROM book WHERE title = '
4       '
5
```

Reply:
Except it might be more robust in theory, but in practice it breaks on our code, because it's getting confused by comments ;).

Reply:
So, better be safe than sorry, and stay with a similar "dumb" split ^^.